e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -4,55 +4,110 @@
 import os
 
 import cdms2
-import MV2
 
 import e3sm_diags
 from e3sm_diags.driver import utils
 from e3sm_diags.logger import custom_logger
-from e3sm_diags.metrics import corr, max_cdms, mean, min_cdms, rmse, std
+from e3sm_diags.metrics import corr, mean, rmse, std
 from e3sm_diags.plot import plot
 
 logger = custom_logger(__name__)
 
 
+def create_and_save_data_and_metrics(parameter, mv1_domain, mv2_domain):
```

Comment:
I didn't make it an internal function, since it could be used by other drivers when we get to refactor those, also lat_lon_land uses it as a base function.

Reply:
Private (internal) functions usually refer to functions that are only called internally by other functions. If it is also intended to be available for the end-user, then it is public.

I don't think it is a big deal in this case though so no need to change it. We can update it later if you want.